packages/medusa-core-utils/src/errors.js

```diff
@@ -5,10 +5,11 @@
 export const MedusaErrorTypes = {
   /** Errors stemming from the database */
   DB_ERROR: "database_error",
+  DUPLICATE_ERROR: "duplicate_error",
```

Comment:
This is appropriate for errors that have to do with duplication in the database. E.g. when trying to insert a customer with an email that already corresponds to a record in the customer table.

Reply:
This is really nice!